packages/app/src/components/OrderDetailsContextMenu.tsx

```diff
@@ -17,12 +18,14 @@ import {
 export const OrderDetailsContextMenu: FC<{ order: Order }> = ({ order }) => {
   const { canUser } = useTokenProvider()
   const [, setLocation] = useLocation()
-
+  const { inventoryModel } = useMarketInventoryModel(order.market?.id)
   const returnableLineItems = useReturnableList(order)
-
+  const orderReturnStockLocation =
+    inventoryModel?.inventory_return_locations ?? []
   const showReturnDropDownItem =
     canUser('create', 'returns') &&
     order.fulfillment_status === 'fulfilled' &&
+    orderReturnStockLocation.length > 0 &&
```

Comment:
shouldn't this be available only if length grater than 1?

Reply:
no, this one is just the control inside the order detail dropdown in order to show or not the "Create return" dropdown item..

Reply:
we added the check over `orderReturnStockLocation.length` as an enforcement.. because in fact if you did not set any return stock location you won't be able to manage returns..